src/ape/managers/chain.py

```diff
@@ -314,7 +312,9 @@ def account_history(self) -> AccountHistory:
         A mapping of transactions from the active session to the account responsible.
         """
         if self.chain_id not in self._account_history_map:
-            history = AccountHistory(self._networks)  # type: ignore
+            AccountHistory._networks = self._networks
+            AccountHistory._converters = self._converters
```

Comment:
Should actually be able to do this injection inside of `__init__`! So you don't have to think about it later.